language/stdlib/transaction_scripts/remove_validator_and_reconfigure_admin.move

```diff
@@ -0,0 +1,21 @@
+script {
+    use 0x1::LibraSystem;
+    use 0x1::SlidingNonce;
+    use 0x1::ValidatorConfig;
+
+    /// Removes a validator from the validator set.
+    /// Fails if the validator_address is not in the validator set.
+    /// Emits a NewEpochEvent.
+    /// Can only be called as an administrative script to trigger waypoints.
+    fun remove_validator_and_reconfigure_admin(
```

Comment:
cc @kchalkias didn't realize this operation would be admin script but this is what you'll need to use